website/content/en/events/2024-cloud-native-sustainability-week.md

```diff
@@ -0,0 +1,50 @@
+---
+title: CNCF Cloud Native Sustainability Week 2024
+linkTitle: Cloud Native Sustainability Week 2024
+exclude_search: true
+description: Cloud Native Sustainability Week is a global event where the CNCF community organizes local meetings around the theme of Cloud Native Sustainability. The Cloud Native Sustainability Week will take place at the beginning week of October 2024.
+weight: 94
+slug: cloud-native-sustainability-week
+aliases:
+  - /cloud-native-sustainability-week
+---
+
+<p class="mt-5 mb-5"><img src="/images/cloud-native-sustainability-week-banner.webp" alt="CNCF Cloud Native Sustainability Week Banner"></p>
```

Comment:
We will have to change the logo here. I should check this with folks who made the logo last year